### DISCUSSION TITLE
Using this plugin with latest version of vite(4.4.5)

### DISCUSSION THREAD

**Discussion by nivethan-me** [crxjs/chrome-extension-tools#808] [Q&A]
Hello everyone,

I hope you are all doing well. I wanted to thank @jacksteamdev & other contributors for the excellent work done on this plugin. After attempting to create a Chrome plugin manually and encountering numerous challenges, I found this plugin to be very helpful.

I am currently in the process of implementing a new Chrome plugin. I have noticed that the plugin documentation states support for Vite 2, while the beta version supports Vite 3 (please correct me if I'm wrong). I am curious whether it is advisable to use this plugin with the latest version of Vite (4.4.5). Will I encounter any production issues in the future if I choose to proceed with the latest Vite version?

Thank you so much for your time.

**Comment by deavial** [comment]
We have been using it with Vite 4 without issue. Vite 5 will break and not compile with a manifest not found error. I can confirm that Vite 4 is safe though.

**Reply by nivethan-me** [reply]
thanks for the info, really appreciate it

**Comment by Toumash** [comment]
Vite 5 just works for react
